GitUI/UserControls/RevisionGrid.cs

```diff
@@ -249,7 +249,7 @@ public Font NormalFont
                 _normalFont = value;
                 MessageDataGridViewColumn.DefaultCellStyle.Font = _normalFont;
                 DateDataGridViewColumn.DefaultCellStyle.Font = _normalFont;
-                _fontOfSHAColumn = new Font("Consolas", _normalFont.SizeInPoints);
+                _fontOfSHAColumn = AppSettings.FixedWidthFont;
```

Comment:
This doesn't look good either, please make it the same font type as everything else

Reply:
This has been hard-coded to be Consolas since the initial implementation (#2660) - do we want to change this?

Reply:
Yes, please. Let's make all columns' text uniform

Reply:
Done.